lineman/app/components/group_page/group_help_card/group_help_card.coffee

```diff
@@ -6,5 +6,11 @@ angular.module('loomioApp').directive 'groupHelpCard', ->
   controller: ($scope, CurrentUser, AppConfig) ->
     $scope.showVideo = AppConfig.loadVideos
 
+    $scope.helpLink = ->
+      if _.contains(['es', 'an', 'ca', 'gl'], CurrentUser.locale)
```

Comment:
I'm also uncomfortable with having hard coded loomio.org urls in here, but not enough to block this going in for now.